instant.ts

```diff
@@ -127,6 +180,9 @@ async function runTests(path: string) {
       chosenPackageIds = Object.keys(allPackages)
     }
 
```

Comment:
When running a `stop` or `destroy` we need to use the reverse order so dependencies are stopped before the dependant packages.

Reply:
addressed https://github.com/openhie/instant/pull/63/commits/725f533b6177a0839464ada7d70db41cd6513dbe